features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And store has "EU VAT" tax rate of 23% for "Clothes" in "EU" zone
+        And store has "No tax" tax rate of 0% for "Clothes" in "Rest of World" zone
```

Comment:
I think we should write this as:

``And store is charging no tax for "Clothes" for the rest of the world`` (we will have a special CATCH_ALL zone type available to avoid the need of adding all countries to "Rest of World" zone.